rotkehlchen/chain/evm/decoding/structures.py

```diff
@@ -82,13 +82,16 @@ class DecodingOutput:
     changed (for example if the user has deposited / withdrawn funds from a curve gauge).
     - reload_decoders can be None in which case nothing happens. Or a set of decoders names for which to reload data. The decoder's name is the class name without the Decoder suffix. For example Eigenlayer for EigenlayerDecoder
     - process_swaps indicates whether there are swaps that need to be converted into EvmSwapEvents.
+    - stop_processing if true will stop processing log events for the transaction and clear
+    any processed event. Used when we want to stop iterating over certain transaction.
```

Comment:
```suggestion
    - stop_processing if true will stop processing log events for the transaction and clear
    any processed events. Used when we want to stop iterating over certain transactions
    because we have determined it's full of unnecessary log events and should all be skipped.
```